### PR TITLE
22/coordinate

### PR DESCRIPTION
**Pull request by JDziewonski98**
# Summary

Created a new dockerized database to support PostGis (a postgres extension which adds a coordinate datatype that can be exported  in GeoJSON format, and can build queries approximately like 'get mural from murals where mural.coordinates within 50km' which I bet will be useful. Updated the model and tests to reflect this. Did some minor refactoring to tests all around so don't worry too much about that. Note the new instructions in readme for getting updated version of database.

## Test Plan

Unit tests

## Related Issues

fixes issue 22, but with the new mandatory coordinate field the controller/service for mural is broken, will branch off and fix for issue 33.

### CODE REVIEW

File: backend/src/models/mural.model.ts [hunk 1]
```diff
@@ -20,6 +20,8 @@ export class Mural extends Model {
   public description!: string;
   public partners!: string[];
   public assistants!: string[];
+  public coordinates!: any;
```
Comment (JDziewonski98) [inline backend/src/models/mural.model.ts:23]: I do not think there is any type support for a PostGis point???

File: backend/src/models/mural.model.ts [hunk 2]
```diff
@@ -79,6 +81,14 @@ Mural.init(
       type: DataTypes.INTEGER,
       references: { model: "boroughs", key: "id" },
     },
+    coordinates: {
+      type: DataTypes.GEOMETRY("POINT", 4326),
```
Comment (tedspare) [inline backend/src/models/mural.model.ts:85]: Would it assign a default SRID without 4326?

File: backend/src/models/mural.model.ts [hunk 3]
```diff
@@ -98,4 +108,7 @@ export interface MuralInterface {
   description?: string;
   partners?: string[];
   assistants?: string[];
+  longitude?: number;
+  latitude?: number;
```
Comment (tedspare) [inline backend/src/models/mural.model.ts:112]: longitude/latitude or coordinates?

File: backend/src/models/tests/mural.test.ts [hunk 4]
```diff
@@ -3,6 +3,21 @@ import { Mural } from "../mural.model";
...
+  name: "testmural",
+  boroughId: "1",
+  artistId: "1",
+  year: 1234,
+  city: "montreal",
+  address: "1234 street",
+  partners: ["partner 1", "partner 2"],
+  coordinates: {
+    type: "Point",
+    coordinates: [-87.123123, 41.232454],
```
Comment (tedspare) [inline backend/src/models/tests/mural.test.ts:18]: nice location hahaha

**Review by tedspare — APPROVED** [review]
LGTM

File: backend/src/models/mural.model.ts (hunk 2 above)
Comment (JDziewonski98) [reply backend/src/models/mural.model.ts:85]: lol honestly the concept of a SRID went over my head. several examples used 4326 tho. https://gis.stackexchange.com/questions/131363/choosing-srid-and-what-is-its-meaning

File: backend/src/models/mural.model.ts (hunk 3 above)
Comment (JDziewonski98) [reply backend/src/models/mural.model.ts:112]: the best practice i saw for creating something with a postgis coordinate field was to pass longitude and latitude in the api call, and then to create a postgis coordinate point using those which is then passed to the Mural.create() call, check the newest mural test case for an example

**Pull request merged**